### ISSUE TITLE
can not use efi in monterey

### ISSUE THREAD

**Issue by golden05** [mendax1234/ThinkpadX390-Opencore-EFI#12]
This EFI now is not use in monterey.

**Comment by mendax1234** [comment]
Yes ，it is not supported yet. l will upgrade the version of OC soon and try it at a later time.

**Comment by mendax1234** [comment]
Now , l have tested the latest EFI on my X390 , l will soon update the related files.

**Comment by golden05** [comment]
I use this efi . but I use my acpi directory. Now , I have upgraded the monterey. but My blueteeth can not work.

**Comment by mendax1234** [comment]
Did you upgrade the itlwm.kext to Monterey version?

**Comment by iriwen** [comment]
I also use this efi，  External monitor does not display through HDMI interface

**Comment by iriwen** [comment]
> I use this efi . but I use my acpi directory. Now , I have upgraded the monterey. but My bluetooth can not work.

I have the same problem

**Comment by mendax1234** [comment]
This is a known issue. l guess that you need to characterize your usb since l use USBinjectall.kext to drive my bluetooth. However, in macOS Monterey, USBinjectall.kext can't be used any longer. So , you need to characterize your USB yourself and maybe the problem will be fixed. 
- l haven't characterized my USB yet , if l have fixed the problem , l will update the related files. If you fix the problem , it is also welcomed to share your files . Thanks!
